Authors: Julien Lavergne <gilir@ubuntu.com>
Description: Revert new button layout.

diff -Naurp obconf.orig/src/preview.c obconf/src/preview.c
--- obconf.orig/src/preview.c	2012-04-23 06:49:15.300632122 +0200
+++ obconf/src/preview.c	2012-04-23 07:05:19.431018918 +0200
@@ -327,28 +327,28 @@ static GdkPixbuf* preview_window(RrTheme
             switch (*layout) {
             case 'D':
                 a = focus ?
-                    theme->btn_desk->a_focused_unpressed :
-                    theme->btn_desk->a_unfocused_unpressed;
+                    theme->a_focused_unpressed_desk :
+                    theme->a_unfocused_unpressed_desk;
                 break;
             case 'S':
                 a = focus ?
-                    theme->btn_shade->a_focused_unpressed :
-                    theme->btn_shade->a_unfocused_unpressed;
+                    theme->a_focused_unpressed_shade :
+                    theme->a_unfocused_unpressed_shade;
                 break;
             case 'I':
                 a = focus ?
-                    theme->btn_iconify->a_focused_unpressed :
-                    theme->btn_iconify->a_unfocused_unpressed;
+                    theme->a_focused_unpressed_iconify :
+                    theme->a_unfocused_unpressed_iconify;
                 break;
             case 'M':
                 a = focus ?
-                    theme->btn_max->a_focused_unpressed :
-                    theme->btn_max->a_unfocused_unpressed;
+                    theme->a_focused_unpressed_max :
+                    theme->a_unfocused_unpressed_max;
                 break;
             case 'C':
                 a = focus ?
-                    theme->btn_close->a_focused_unpressed :
-                    theme->btn_close->a_unfocused_unpressed;
+                    theme->a_focused_unpressed_close :
+                    theme->a_unfocused_unpressed_close;
                 break;
             default:
                 continue;
